pkg/manifests/manifests.go

```diff
@@ -1707,6 +1709,9 @@ func (f *Factory) PrometheusUserWorkload(grpcTLS *v1.Secret, trustedCABundleCM *
 		}
 	}
 
+	// Enable experimental additional scrape metrics feature
+	p.Spec.EnableFeatures = append(p.Spec.EnableFeatures, EnableFeatureExtraScrapeMetricsString)
```

Comment:
We could also set this in the YAML manifest directly?

Reply:
right we are not giving it as option so adding in asset file makes sense. I updated